### PR TITLE
Cause lists

### PR DESCRIPTION
**Pull request by actlikewill**
image

closes https://github.com/laws-africa/peachjam/issues/2094

### CODE REVIEW

File: peachjam/models/judgment.py [hunk 1]
```diff
@@ -539,3 +539,21 @@ def __str__(self):
...
+        on_delete=models.PROTECT,
+        null=True,
+        verbose_name=_("court"),
+        related_name="causelists",
+    )
+    judges = models.ManyToManyField(Judge, blank=True, verbose_name=_("judges"))
+
+    def pre_save(self):
+        self.frbr_uri_doctype = "doc"
+        self.doc_type = "causelist"
```
Comment (actlikewill) [inline peachjam/models/judgment.py:558]: @longhotsummer the frbr_uri  is not getting assigned correctly, should it be `/akn/doc/causelist/causelist/etc`? should the court be included?

File: peachjam/models/judgment.py [hunk 2]
```diff
@@ -115,6 +115,23 @@ def save(self, *args, **kwargs):
...
+        "peachjam.EntityProfile", verbose_name=_("profile")
+    )
+    order = models.IntegerField(_("order"), null=True, blank=True)
+
+    def __str__(self):
+        return self.name
+
+    def save(self, *args, **kwargs):
+        if not self.code:
+            self.code = f"{slugify(self.name)}"
```
Comment (longhotsummer) [inline peachjam/models/judgment.py:131]:
```suggestion
            self.code = slugify(self.name)"
```

Comment (actlikewill) [reply peachjam/models/judgment.py]: fixed

**Comment by actlikewill** [comment]
@longhotsummer I've added the document nature to the list filters and fixed the causelist titles to use the court registry by default and show the date in full
image

**Review by longhotsummer — APPROVED** [review]

**Pull request merged**